scheduler/src/cook/mesos/api.clj

```diff
@@ -341,8 +341,7 @@
   (-> JobRequestMap
       (dissoc (s/optional-key :group))
       (dissoc (s/optional-key :status))
-      (merge {:framework-id (s/maybe s/Str)
-              :retries-remaining NonNegInt
+      (merge {:retries-remaining NonNegInt
```

Comment:
Discussed above, I don't think we should remove this, even if it's currently broken.